tests/ui/consts/miri_unleashed/mutable_references.rs

```diff
@@ -4,29 +4,27 @@ use std::cell::UnsafeCell;
 
 // a test demonstrating what things we could allow with a smarter const qualification
 
-// this is fine because is not possible to mutate through an immutable reference.
 static FOO: &&mut u32 = &&mut 42;
+//~^ ERROR encountered mutable pointer in final value of static
 
-// this is fine because accessing an immutable static `BAR` is equivalent to accessing `*&BAR`
-// which puts the mutable reference behind an immutable one.
 static BAR: &mut () = &mut ();
+//~^ ERROR encountered mutable pointer in final value of static
 
 struct Foo<T>(T);
 
-// this is fine for the same reason as `BAR`.
 static BOO: &mut Foo<()> = &mut Foo(());
+//~^ ERROR encountered mutable pointer in final value of static
 
 // interior mutability is fine
```

Comment:
outdated comment, this test generally feels somewhat useless rn, given that its behavior doesn't depend on `-Zunleash-the-miri-inside-of-you` afaict?

Reply:
stderr has a bunch of "skipping check that does not even have a feature gate", so I think it does depend on the unleash flag.